File: panorama/src/main/kotlin/com/isycat/dotaaddon/panorama/unitCardWrapperCardPanel.kt
package com.isycat.dotaaddon.panorama

import com.isycat.dota.panorama.*

fun unitCardWrapperCardPanel() =
    Panel(classes = "UnitCardZXC", hittest = false) {
        Panel(classes = "CardTransformWrapper", id = "CardTransformWrapperX", hittest = false) {
            Label(id = "CardQuantity", hittest = false)
            Button(classes = "ShopItem", id = "ShopItemX")
        }
    }

File: content/panorama/scripts/custom_game/unitCardWrapperCardPanel.js
// package: com.isycat.dotaaddon.panorama

function unitCardWrapperCardPanel() {
    return new Panel("UnitCardZXC", false, () => {
        return new Panel("CardTransformWrapper", "CardTransformWrapperX", false, () => {
            new Label("CardQuantity", false);
            return new Button("ShopItem", "ShopItemX");
        });
    });
}

